libcaf_core/caf/event_based_mail.test.cpp

```diff
@@ -1159,13 +1151,9 @@ TEST("send fan_out_request messages that return two swapped values using typed "
             .fan_out_request(workers, infinite, policy::select_all_tag)
             .as_observable()
             .do_on_error([err](const error& x) { *err = x; })
-            .for_each(
-              [swapped_values](std::vector<caf::cow_tuple<int, int>> results) {
-                swapped_values->clear();
-                for (auto result : results) {
-                  swapped_values->emplace_back(get<0>(result), get<1>(result));
-                }
-              });
+            .for_each([swapped_values](const caf::cow_tuple<int, int>& result) {
```

Comment:
An interface question: now that we no longer have a `vector`, would a lambda accepting two `int`s make more sense? 
```suggestion
            .for_each([swapped_values](int first, int second) {
```
I saw that the `select_any` uses cow_tuples as well, so I suppose that's how flows transfer them, and it would be a breaking change. Just want to hear your comment on this.